src/multivariate/mvnormal.jl

```diff
@@ -65,6 +65,13 @@ const ZeroMeanIsoNormal{Axes}  = MvNormal{ScalMat,  Zeros{Float64,1,Axes}}
 const ZeroMeanDiagNormal{Axes} = MvNormal{PDiagMat, Zeros{Float64,1,Axes}}
 const ZeroMeanFullNormal{Axes} = MvNormal{PDMat,    Zeros{Float64,1,Axes}}
 ```
+
+Multivariate normal distributions support affine transformations:
+```julia
+d = MvNormal(μ, Σ)
```

Comment:
Maybe give more explicit names to c and B to indicate these are constant vectors and matrices respectively

Reply:
I think this is pretty obvious from the context, but if you have a suggestion I am happy to change them.